src/main/java/seedu/address/model/person/Tutorial.java

```diff
@@ -5,12 +5,15 @@
 
 /**
  * Represents a Student's tutorial in StudentConnect.
- * Guarantees: immutable;
+ * Guarantees: immutable;  is valid as declared in {@link #isValidTutorial(String)}
  */
 public class Tutorial {
 
     public static final String MESSAGE_CONSTRAINTS = "Tutorials should be integers between 01 and 22, "
-                                                        + "separated by a comma.";
+                                                        + "separated by a space.\n"
+                                                        + "Constraint: Tutorial should only contain numbers, "
+                                                        + "and it should be 2 digit long between 01 and 22";
```

Comment:
This line is a repeat of line 12. Can you rephrase the message and follow how the other messages has been written for the other classes thanks!

Reply:
Changed it to "Tutorials should be 2-digit numbers between 01 and 22, separated by spaces." Hope that's okay!

Reply:
Yup thank you!